dockerfiles/squid/Dockerfile

```
@@ -1,6 +1,6 @@
 FROM alpine:3.8
 
-RUN apk add --no-cache squid=3.5.27-r4
+RUN apk add --no-cache squid=4.11-r0
```

```suggestion
RUN apk add --no-cache squid=4.10-r0
```
It works if we use alpine:3.11.

---

Fixed.